homeassistant/components/waze_travel_time/sensor.py

```diff
@@ -185,14 +185,14 @@ def update(self):
                 )
                 routes = params.calc_all_routes_info(real_time=realtime)
 
-                if incl_filter is not None:
+                if incl_filter not in (None, ""):
```

Comment:
```suggestion
                if incl_filter not in {None, ""}:
```

Reply:
Thanks. Can you tell me what the advantages of your approach are?

Reply:
`in` is slightly more optimised with sets `{x, y}` than with tuples `(x, y)` or lists `[x, y]`